components/style/properties/data.py

```diff
@@ -41,7 +44,18 @@ def values_for(self, product):
             raise Exception("Bad product: " + product)
 
     def gecko_constant(self, value):
-        return self.gecko_constant_prefix + "_" + value.replace("-moz-", "").replace("-", "_").upper()
+        if self.gecko_enum_prefix:
+            if self.enum_replace and value in self.enum_replace:
+                return self.gecko_enum_prefix + "::" + self.enum_replace[value]
```

Comment:
Do we expect to need enum_replace for anything other than `None`?  If not, then I'd be happy to just to hard code that in here until we solve that on the Gecko side.  As we convert more macros in nsStyleConsts.h we may as well make them use names that `gecko_constan` expects here.